cocos/renderer/backend/opengl/BufferGL.h

```diff
@@ -16,24 +16,22 @@ class BufferGL : public Buffer
     
     virtual void updateData(void* data, unsigned int size) override;
     virtual void updateSubData(void* data, unsigned int offset, unsigned int size) override;
-    
+    virtual void usingDefaultStoredData(bool needDefaultStoredData) override ;
+
     inline GLuint getHandler() const { return _buffer; }
 
-private:
```

Comment:
why remove it?

Reply:
Yes, private is needed and its a mistake, I will fix it.